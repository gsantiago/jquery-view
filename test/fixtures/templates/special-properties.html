<div>
  {{user}}
  {{$index}}
  {{$key}}
  {{$total}}
  {{$first}}
  {{$middle}}
  {{$last}}
  {{$even}}
  {{$odd}}
  <p :repeat="user in users">
    {{user}}
    {{$index}}
    {{$key}}
    {{$total}}
    {{$first}}
    {{$middle}}
    {{$last}}
    {{$even}}
    {{$odd}}
  </p>
  {{user}}
  {{$index}}
  {{$key}}
  {{$total}}
  {{$first}}
  {{$middle}}
  {{$last}}
  {{$even}}
  {{$odd}}
</div>