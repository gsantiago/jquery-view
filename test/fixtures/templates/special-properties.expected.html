<div>
  Initial user
  Initial index
  Initial key
  Initial total
  Initial first
  Initial middle
  Initial last
  Initial even
  Initial odd
  <p>
    Guilherme
    0
    0
    3
    true
    false
    false
    true
    false
  </p><p>
    Junin
    1
    1
    3
    false
    true
    false
    false
    true
  </p><p>
    Zilla
    2
    2
    3
    false
    false
    true
    true
    false
  </p>
  Initial user
  Initial index
  Initial key
  Initial total
  Initial first
  Initial middle
  Initial last
  Initial even
  Initial odd
</div>